Dewikification - Migrate event pages from wiki to HTML

ks129 opened the pull request:
Closes #386 

Migrated all current event pages to dewikified event pages system.

ks129 committed:
Create upcoming event sidebar block

ks129 committed:
Create events list (year calendar) sidebar block

ks129 committed:
Finish events index page

ks129 committed:
Update Code Jam 7 page to match with latest revision in wiki

ks129 committed:
Migrate Code Jam 1 page

ks129 committed:
Make upcoming event sidebar block image to link

ks129 committed:
Create upcoming code jam sidebar block

ks129 committed:
Create previous code jams sidebar block

ks129 committed:
Create code jams useful information sidebar block

ks129 committed:
Migrate code jams page from wiki to HTML

ks129 committed:
Make useful information sidebar block name clearer

ks129 committed:
Migrate Code Jams Git using page

ks129 committed:
Migrate Code Jams judging page from wiki to HTML

ks129 committed:
Migrate Code Jams PR opening page from wiki to HTML

ks129 committed:
Use file instead directory for code jam 1

ks129 committed:
Add highlight.js to event pages

ks129 committed:
Migrate code jam 2 page from wiki to HTML

ks129 committed:
Migrate code jam 3 page from wiki to HTML

ks129 committed:
Migrate code jam 4 page from wiki to HTML

ks129 committed:
Migrate code jam 5 page from wiki to HTML

ks129 committed:
Create siderbar item for code jam 6 page

ks129 committed:
Migrate code jam 6 page from wiki to HTML

ks129 committed:
Use h2 instead bigger h-s and add classes

ks129 committed:
Migrate code jam 6 rules page from wiki to HTML

ks129 committed:
Create sidebar block for game jam 2020

ks129 committed:
Create general game jams page (not ready)

ks129 committed:
Migrate game jam 2020 page from wiki to HTML

MarkKoz reviewed (commented):
I can't figure out how to get the the code jam or game jam pages. I've only managed to navigate to `http://localhost:8000/events/` and the links in the event calendar are all broken.

MarkKoz reviewed (changes requested):
The code block here has looks like its embedded within a light grey rectangle. That shouldn't happen

![bild](https://user-images.githubusercontent.com/1515135/111673931-b425e480-87d8-11eb-938e-b935a796376c.png)

The sidebar on the right does not appear in code jams 1-5 despite the page content still being narrower to accommodate for the sidebar.

kosayoda committed:
Fix event content columns when there is no sidebar.

Prevents page contents of events pages from being narrower when there is
no sidebar to fill the space.

kosayoda committed:
Style <pre> tag backgrounds the same as <code>.

This makes it consistent with the django-wiki version of the code
blocks, which looks neater.

MarkKoz reviewed (approved):
Everything looks to be in order now.
